lib/msal-core/src/ScopeSet.ts

```diff
@@ -32,8 +33,9 @@ export class ScopeSet {
      * @param scopes
      */
     static containsScope(cachedScopes: Array<string>, scopes: Array<string>): boolean {
-        cachedScopes = this.convertToLowerCase(cachedScopes);
-        return scopes.every((value: any): boolean => cachedScopes.indexOf(value.toString().toLowerCase()) >= 0);
+        const convertedCachedScopes = this.trimAndConvertToLowerCase([...cachedScopes]);
+        const requestScopes = this.trimAndConvertToLowerCase([...scopes]);
+        return requestScopes.every((value: any): boolean => convertedCachedScopes.indexOf(value.toString().toLowerCase()) >= 0);
```

Comment:
is every supported in all browsers @jasonnutter

Reply:
Yes.